### PR TITLE
setup inpycon 2017

### PR DESCRIPTION
**Pull request by bhanuvrat**
update nginx conf to setup inpycon 2017 website.

### CODE REVIEW

File: salt/roots/inpycon/in.pycon2017.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+location ~ ^/2016/(.*)$ {
+   alias /opt/inpycon2016/$1;
+}
+
+location = / {
+    rewrite "^.*$" /2016/ redirect;
```
Comment (vigneshsarma) [inline salt/roots/inpycon/in.pycon2017.conf]: shouldn't this be 2017?

Comment (bhanuvrat) [reply salt/roots/inpycon/in.pycon2017.conf]: ouch .. yes of course.
skipped that.

Comment (bhanuvrat) [reply salt/roots/inpycon/in.pycon2017.conf]: fixed and force pushed.
@vigneshsarma take a look.

**Comment by bhanuvrat** [comment]
@vigneshsarma this is cool now?

File: salt/roots/inpycon/in.pycon2017.conf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+location ~ ^/2016/(.*)$ {
+   alias /opt/inpycon2017/$1;
```
Comment (vigneshsarma) [inline salt/roots/inpycon/in.pycon2017.conf]: shouldn't this also be 2017?

**Comment by bhanuvrat** [comment]
@vigneshsarma please check once more.

**Pull request merged**